From ac774e87f52b5b16b7cf6c2ea452498b53d067f2 Mon Sep 17 00:00:00 2001
From: "Thakur, Santosh Krx" <santosh.krx.thakur@intel.com>
Date: Thu, 4 Feb 2016 18:08:03 +0530
Subject: [PATCH 19/24] ASoC: Intel: klockwork error in Audio Driver

This patch returns "0" from the function
having return type as integer.

Change-Id: Icbc48d670650d56b5d45528463fb8cba7f93b1c2
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-18672
Signed-off-by: Thakur, Santosh Krx <santosh.krx.thakur@intel.com>
Reviewed-on: https://android.intel.com:443/466513
---
 sound/soc/soc-dapm.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/soc-dapm.c b/sound/soc/soc-dapm.c
index a585222..85a6607 100644
--- a/sound/soc/soc-dapm.c
+++ b/sound/soc/soc-dapm.c
@@ -3458,6 +3458,7 @@ static int snd_soc_get_hw_params(struct snd_pcm_hw_params *params,
 		= config->channels_min;
 	hw_param_interval(params, SNDRV_PCM_HW_PARAM_CHANNELS)->max
 		= config->channels_max;
+	return 0;
 }
 
 static int snd_soc_dai_link_event(struct snd_soc_dapm_widget *w,
-- 
1.9.1

